src/content/infanterie/mg-schuetze.svx

```diff
@@ -1,41 +1,57 @@
+
+<Slideshow
+images={[m249,mg4]}
+captions={["M249, USA", "MG4, Deutschland"]}
+/>
 
-### Leichte Maschinengewehre
+### Mittlere Maschinengewehre (<Tooltip text="Medium Machine Gun">MMG</Tooltip>)
 
-Zu den leichten Maschinengewehren zählen unter anderem das deutsche MG5, das amerikanische M249 und das russische PKM. Diese Waffen können in der Hand mitgeführt und durch einen einzigen Soldaten bedient werden. Leichte Maschinengewehre verschießen Kaliber wie 5,56x45 mm oder 7,62x51 mm NATO und haben eine hohe <Tooltip text="Schussfrequenz oder auch Schusskadenz">Feuerrate</Tooltip>.
+Zu den mittleren Maschinengewehren zählen wir schwere Handfeuerwaffen, die durch ein [MG-Buddyteam](/ausbildung/truppenteile#buddyteam) eingesetzt werden. Das Buddyteam besteht aus dem MG-Schützen und einem MG-Assistenten, der zusätzlich Munition für das MMG mitführt. Im Gegensatz zu LMG-Schützen agieren MMG-Schützen also nicht autark. MMG verschießen im Durchschnitt den Munitionstyp 7,62x51mm NATO und haben damit eine höhere Durchschlagskraft als LMG und eine effektive Reichweite von 800-1800 Metern.
 
-### Schwere Maschinengewehre
+### Schwere Maschinengewehre (<Tooltip text="Heavy Machine Gun">HMG</Tooltip>)
 
-Zu den schweren Maschinengewehren zählen unter anderem das amerikanische M2 und das russische DShK. Diese Waffen werden aufgrund ihres Gewichtes hauptsächlich stationär auf einem Dreibein oder auf dem Dach eines Fahrzeugs montiert eingesetzt. Alternativ müssten sie demontiert in zwei Rucksäcken durch das Feld getragen werden, wodurch sie sich bei der Infanterie eher zu defensiven Zwecken eignen. Diese Waffen verschießen Kaliber wie 12,7x99 mm NATO oder 12,7x108 mm und haben eine niedrigere Feuerrate als leichte Maschinengewehre, dafür aber eine höhere effektive Reichweite und hohe Durchschlagskraft, die zur Bekämpfung von leichtgepanzerten Fahrzeugen eingesetzt werden kann.
+Zu den schweren Maschinengewehren zählen grundsätzlich stationäre oder auf Fahrzeugen montierte Waffen, die primär zu defensiven Zwecken eingesetzt werden. Diese können demontiert in zwei Rucksäcken von Infanteristen mitgeführt werden, was aufgrund der Vorteile von LMG und MMG gegenüber HMG allerdings selten vorkommt. Diese Waffen verschießen Munitionstypen wie 12,7x99mm NATO oder 12,7x108mm und haben damit eine überwältigende Durchschlagskraft und eine effektive Reichweite von bis zu 2000 Metern. Mit ihrer Durchschlagskraft eignen sich HMG zur Bekämpfung von leichtgepanzerten Fahrzeugen.
```

Comment:
```suggestion
Zu den schweren Maschinengewehren zählen grundsätzlich stationäre oder auf Fahrzeugen montierte Waffen, die primär zu defensiven Zwecken eingesetzt werden. Diese können demontiert in zwei Rucksäcken von Infanteristen mitgeführt werden, was aufgrund der Vorteile von LMG und MMG gegenüber HMG allerdings selten vorkommt. Diese Waffen verschießen Munitionstypen wie 12,7x99 mm oder 12,7x108 mm NATO und haben damit eine überwältigende Durchschlagskraft und eine effektive Reichweite von bis zu 2000 Metern. Mit ihrer Durchschlagskraft eignen sich HMG zur Bekämpfung von leichtgepanzerten Fahrzeugen.
```

Reply:
I think you accidentaly moved the "NATO". 12,7x99 mm is the NATO .50 ammo type, 12,7x108 mm is commonly used by russian weapons. Apart from that, I'd commit these changes.

Reply:
Oh I thought these were both NATO ammunition types and thought it helped with the sentence flow. 
In that case that needs to be reverted. I'm updating my suggestion.

```suggestion
Zu den schweren Maschinengewehren zählen grundsätzlich stationäre oder auf Fahrzeugen montierte Waffen, die primär zu defensiven Zwecken eingesetzt werden. Diese können demontiert in zwei Rucksäcken von Infanteristen mitgeführt werden, was aufgrund der Vorteile von LMG und MMG gegenüber HMG allerdings selten vorkommt. Diese Waffen verschießen Munitionstypen wie 12,7x99 mm NATO oder 12,7x108 mm und haben damit eine überwältigende Durchschlagskraft und eine effektive Reichweite von bis zu 2000 Metern. Mit ihrer Durchschlagskraft eignen sich HMG zur Bekämpfung von leichtgepanzerten Fahrzeugen.
```